ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NTP_TIME/tasks/main.yml

```diff
@@ -16,6 +16,7 @@
   when:
     - ansible_distribution != "CentOS" and ansible_distribution != "RedHat"
     - ansible_distribution != "FreeBSD"
+    - ansible_architecture != "riscv64"
```

Comment:
Does this role not work either?

Reply:
there is no `systemd-timesyncd` service on the debain riscv system I'm using. It could be a Debian thing in general but didn't want to make any assumtions

Reply:
@Willsparker Can you confirm whether this is the case on the other Debian systems you're testing with?

Reply:
```
$ sudo service systemd-timesyncd status
sudo: unable to resolve host debian: Name or service not known
● systemd-timesyncd.service - Network Time Synchronization
     Loaded: loaded (/lib/systemd/system/systemd-timesyncd.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2020-08-25 08:57:45 UTC; 3h 17min ago
       Docs: man:systemd-timesyncd.service(8)
   Main PID: 227 (systemd-timesyn)
     Status: "Initial synchronization to time server 171.66.97.126:123 (0.debian.pool.ntp.org)."
      Tasks: 2 (limit: 6940)
     Memory: 1.3M
     CGroup: /system.slice/systemd-timesyncd.service
             └─227 /lib/systemd/systemd-timesyncd
```
It's there for me on Debian Bullseye

Reply:
There is a `systemd-timesyncd` package available in the RISC-V Debian10 repository so we should ensure that is installed as appropriate instead of excluding on this architecture

Reply:
@gdams Can you reinstate this based on my previous comment?

Reply:
updated